spec/factories/users.rb

```diff
@@ -32,6 +32,7 @@
 
     after(:create) do |user|
       user.is_admin = false
+      user.is_disabled = false
```

Comment:
Because of https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L18 and https://github.com/openSUSE/osem/blob/master/app/models/user.rb#L200-L204 we need the after block for the is_admin attribute, the is_disabled attribute can be set normally